expression/builtin_op.go

```diff
@@ -815,15 +815,15 @@ type unaryMinusFunctionClass struct {
 func (c *unaryMinusFunctionClass) handleIntOverflow(arg *Constant) (overflow bool) {
 	if mysql.HasUnsignedFlag(arg.GetType().GetFlag()) {
 		uval := arg.Value.GetUint64()
-		// -math.MinInt64 is 9223372036854775808, so if uval is more than 9223372036854775808, like
-		// 9223372036854775809, -9223372036854775809 is less than math.MinInt64, overflow occurs.
+		// -mathutil.MinInt64 is 9223372036854775808, so if uval is more than 9223372036854775808, like
+		// 9223372036854775809, -9223372036854775809 is less than mathutil.MinInt64, overflow occurs.
 		if uval > uint64(-math.MinInt64) {
 			return true
 		}
 	} else {
 		val := arg.Value.GetInt64()
-		// The math.MinInt64 is -9223372036854775808, the math.MaxInt64 is 9223372036854775807,
-		// which is less than abs(-9223372036854775808). When val == math.MinInt64, overflow occurs.
+		// The mathutil.MinInt64 is -9223372036854775808, the mathutil.MaxInt64 is 9223372036854775807,
+		// which is less than abs(-9223372036854775808). When val == mathutil.MinInt64, overflow occurs.
```

Comment:
I think you can revert these changes, since the comments are most likely still correct.